x/tunnel/types/params.go

```diff
@@ -3,19 +3,23 @@ package types
 import (
 	"fmt"
 
+	channeltypes "github.com/cosmos/ibc-go/v8/modules/core/04-channel/types"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
 var (
 	// Each value below is the default value for each parameter when generating the default
 	// genesis file. See comments in types.proto for explanation for each parameter.
-	DefaultMinInterval     = uint64(60)
-	DefaultMaxInterval     = uint64(3600)
-	DefaultMinDeviationBPS = uint64(50)
-	DefaultMaxDeviationBPS = uint64(3000)
-	DefaultMinDeposit      = sdk.NewCoins(sdk.NewInt64Coin("uband", 1_000_000_000))
-	DefaultMaxSignals      = uint64(25)
-	DefaultBasePacketFee   = sdk.NewCoins(sdk.NewInt64Coin("uband", 500))
+	DefaultMinInterval               = uint64(60)
+	DefaultMaxInterval               = uint64(3600)
+	DefaultMinDeviationBPS           = uint64(50)
+	DefaultMaxDeviationBPS           = uint64(3000)
+	DefaultMinDeposit                = sdk.NewCoins(sdk.NewInt64Coin("uband", 1_000_000_000))
+	DefaultMaxSignals                = uint64(25)
+	DefaultBasePacketFee             = sdk.NewCoins(sdk.NewInt64Coin("uband", 500))
+	DefaultRouterIBCChannel          = "channel-0"
+	DefaultRouterIntegrationContract = "router17c2txg2px6vna8a6v4ql4eh4ruvprerhytxvwt2ugp4qr473pajsyj9pgm"
```

Comment:
- remove default for channel
- contract will be the same for both testnet and mainnet? if not, please remove it.